src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+public class Parser {
+    private String command;
+    private String[] twoPart;
+    private String[] time;
+
+    public Parser(String command) {
+        this.command = command;
+        this.twoPart = this.command.split(" ", 2);
+    }
+
+    public String getCommand() {
+        return this.command;
+    }
+
+    public String firstPart() {
+        return this.twoPart[0];
+    }
+
+    public int secondPartInInt() throws DukeException {
```

Comment:
Method names should be verbs. So this would be better named as 'getSecondPartInInt'.